File: RobotLib/src/main/java/com/acmerobotics/library/cameraoverlay/message/PongResponse.java
package com.acmerobotics.library.cameraoverlay.message;

public class PongResponse extends Response {
    public PongResponse() {
        super(MessageType.PONG);
    }

    @Override
    public void set(Object value) {

    }

    @Override
    public Object get() {
        return null;
    }

    @Override
    public byte[] payloadToByteArray() {
        return new byte[0];
    }

    @Override
    public void payloadFromByteArray(byte[] bytes) {

    }
}

File: CameraOverlay/src/main/java/com/acmerobotics/cameraoverlay/message/PongResponse.kt
package com.acmerobotics.cameraoverlay.message

class PongResponse: Response<Unit>(MessageType.PONG) {
    override fun set(value: Unit) {}
    override fun get() {}

    override fun payloadToByteArray(): ByteArray = ByteArray(0)

    override fun payloadFromByteArray(byteArray: ByteArray) {}

}